init_node.sh

```diff
@@ -124,7 +124,7 @@ for ((i = 0; i < N; i++))
       sed -i 's/addr_book_strict = true/addr_book_strict = false/g' ${CHAIN_DIR}/config/config.toml  # for local test
       sed -i 's/allow_duplicate_ip = false/allow_duplicate_ip = true/g' ${CHAIN_DIR}/config/config.toml  # allow duplicated ip
 
-      sed -i 's#'"${MEMO}"'#'"${MEMO_SPLIT[1]}"':'"${P2P_PORT}"'#g' ${CHAIN_0_DIR}/config/config.toml  # change port of persistent_peers
+      sed -i 's#'"${MEMO}"'#'"${MEMO_SPLIT[0]}"':'"${P2P_PORT}"'#g' ${CHAIN_0_DIR}/config/config.toml  # change port of persistent_peers
```

Comment:
Did you work well?
When I check it, it is ok if index is `1`. But the value of `0` index is empty. 

```
MEMO_SPLIT=(`echo ${MEMO} | tr ":" "\n"`)
echo "MEMO: ${MEMO}"
echo "MEMO_SPLIT: ${MEMO_SPLIT}"
echo "MEMO_SPLIT[0]: ${MEMO_SPLIT[0]}"
echo "MEMO_SPLIT[1]: ${MEMO_SPLIT[1]}"
echo "MEMO_SPLIT[2]: ${MEMO_SPLIT[2]}"
~~~~
output
MEMO_SPLIT: f162f9e197cb926ad99a10c4ffc295c996e20f9d@192.168.0.53 26656
MEMO_SPLIT[0]: 
MEMO_SPLIT[1]: f162f9e197cb926ad99a10c4ffc295c996e20f9d@192.168.0.53
MEMO_SPLIT[2]: 26656
```